contrib/create-waku/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "create-waku",
   "version": "0.4.1",
-  "bin": "./cli.js"
+  "bin": "./cli.js",
+  "files": [
+    "cli.js"
+  ],
+  "dependencies": {
+    "@clack/core": "=0.3.2",
```

Comment:
I've never seen this format.
What's the difference from `"0.3.2"`?
ref: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#dependencies

Reply:
This is my mistake. But I works correctly🤔️